src/basic/glyph-util.c

```diff
@@ -153,6 +154,7 @@ const char* special_glyph_full(SpecialGlyph code, bool force_utf) {
                         [SPECIAL_GLYPH_SUPERHERO]               = u8"🦸",
                 },
         };
+        REENABLE_WARNING;
```

Comment:
hmm, maybe we should switch to not prefix those strings with u8 anymore? why are we doing that again?

Reply:
But, without that, maybe we need to write string like before PR #22711 . That's hard to read...

Reply:
so, hmm, you are saying that we have no way to write literal utf8 in native sign of char? that kinda sucks...

maybe we should solve this via a macro?

i.e.:

```c
#define UTF8(s) ((const char*) (u8"" s))
```

i didn't test this, but with that we could do:

```c
[SPECIAL_FOO] = UTF8("🦸"),
```

couldn't we, and it would be fully correct on all C releases right now? and quite readable too i'd say?

Reply:
I tried something similar but that did not worked. I will try the suggested way later.